packages/app/MANIFEST.in

```diff
@@ -1,5 +1,4 @@
 include *.rst *.txt LICENSE */py.typed
-include galaxy/carbon_emissions/carbon_intensity.csv
```

Comment:
Bad merge?

Reply:
These were removed on purpose to cause test failures, but the weird thing is, the tests are passing. `carbon_intensity.csv` was removed to simulate this issue: https://github.com/galaxyproject/galaxy/pull/17748
but the relevant test is not failing as I mentioned in: https://github.com/galaxyproject/galaxy/pull/17747#issuecomment-2018270355
(it's finally failing now, but for other, unrelated reasons)

Reply:
Ah, sorry, I've missed that. The `test_galaxy_packages` tests fail for the right reason now that https://github.com/galaxyproject/galaxy/pull/17991 has been merged.

Reply:
I rebased just in case, but I don't think that change affects this? This builds the wheels locally, and forces tpv to use those wheels: https://github.com/galaxyproject/galaxy/actions/runs/8879844569/job/24378560228?pr=17747#step:6:971

I would therefore have expected the missing file to cause the expected failure, but it is not. So something else must be going on?

Reply:
I think this was due to using the wrong `pytest.ini` in package tests, see https://github.com/galaxyproject/galaxy/pull/18301 for a fix.